presto-main/src/main/java/io/prestosql/operator/aggregation/builder/InMemoryHashAggregationBuilder.java

```diff
@@ -178,7 +152,7 @@ public void close()
     @Override
     public void updateMemory()
     {
-        updateMemoryWithYieldInfo();
+        updateMemory.update();
```

Comment:
`updateMemory.update(getSizeInMemory());`

Reply:
Can't do that, sorry.